torch/_torch_docs.py

```diff
@@ -515,6 +515,11 @@ def merge_dicts(*dicts):
 For inputs of type `FloatTensor` or `DoubleTensor`, arguments :attr:`beta` and
 :attr:`alpha` must be real numbers, otherwise they should be integers
 
+.. warning::
+    This function is deprecated and may be removed in a future release.
+    And it can be implemented by calling :func:`torch.outer`
```

Comment:
"... in a future release. It can be implemented using :func:\`torch.outer\` as \`\`alpha * torch.outer(vec1, vec2) + beta * input\`\` when :attr:\`beta\` is not zero, and as \`\`alpha * torch.outer(vec1, vec2)\`\` when :attr:\`beta\` is zero."

Reply:
Addressed.